definitions/infra-apacheserver/golden_metrics.yml

```diff
@@ -3,12 +3,22 @@ requestsPerSecond:
   query:
     select: average(net.requestsPerSecond)
     from: ApacheSample
-    facet: entityName
```

Comment:
This will default now to `entity.name` is that expected? Same for all the others